### PR TITLE
Import tables from Mongo

### PR DESCRIPTION
**Pull request by darroyocazorla**

### CODE REVIEW

File: mongodb/src/main/scala/com/stratio/crossdata/connector/mongodb/DefaultSource.scala [hunk 1]
```diff
@@ -91,26 +142,29 @@ class DefaultSource extends RelationProvider with SchemaRelationProvider with Cr
...
+      //TODO improve code
+      case (properties, Credentials) =>
 
-    val finalMap = (properties /: optionalProperties){    //TODO improve code
-      case (properties,Credentials) =>
         /** We will assume credentials are provided like 'user,database,password;user,database,password;...' */
         val credentialInput = parameters.getOrElse(Credentials, " ")
-        if(credentialInput.compareTo(" ")!=0){
-          val credentials= credentialInput
+        if (credentialInput.compareTo(" ") != 0) {
```
Comment (miguel0afd) [inline mongodb/src/main/scala/com/stratio/crossdata/connector/mongodb/DefaultSource.scala]: Please, use isEmpty instead

File: mongodb/src/main/scala/com/stratio/crossdata/connector/mongodb/DefaultSource.scala [hunk 2]
```diff
@@ -91,26 +142,29 @@ class DefaultSource extends RelationProvider with SchemaRelationProvider with Cr
...
             .map(credentials => MongodbCredentials(credentials(0), credentials(1), credentials(2).toCharArray))
           properties.+(Credentials -> credentials)
         } else properties
-      case (properties,SSLOptions) =>
+      case (properties, SSLOptions) =>
+
         /** We will assume ssloptions are provided like '/path/keystorefile,keystorepassword,/path/truststorefile,truststorepassword' */
         val ssloptionInput = parameters.getOrElse(SSLOptions, " ")
-        if(ssloptionInput.compareTo(" ")!=0) {
+        if (ssloptionInput.compareTo(" ") != 0) {
```
Comment (miguel0afd) [inline mongodb/src/main/scala/com/stratio/crossdata/connector/mongodb/DefaultSource.scala]: Please, use isEmpty instead

Comment (darroyocazorla) [inline mongodb/src/main/scala/com/stratio/crossdata/connector/mongodb/DefaultSource.scala]: It's not my code. I proposed to use isEmpty but we are going to replace with the new Mongo code as soon as posible. Anyway, it's not related to this PR.

File: mongodb/src/main/scala/com/stratio/crossdata/connector/mongodb/DefaultSource.scala (hunk 1 above)
Comment (darroyocazorla) [inline mongodb/src/main/scala/com/stratio/crossdata/connector/mongodb/DefaultSource.scala]: See the comment below

**Comment by miguel0afd** [comment]
LGTM +1

**Comment by jjlopezm** [comment]
+1

**Pull request merged**